test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+rm -rf test
+mkdir -p test
+mkdir -p test/mount_point
+mkdir -p test/gitfs_owned_repo
+mkdir -p test/origin.git
+cat > test/.gitconfig <<EOF 
+[user] 
+       name = GitFs
+       email = gitfs@gitfs.com\n
+
+EOF
+
+ROOT=`pwd`
+
+cd test/origin.git;\
+    HOME=../../test git init --bare .;\
+    cd ../../;\
```

Comment:
Now that I can login to drone I see the steps i assumed were part of the drone config are actually part of the makefile and i just duplicated them in test.sh :|